front/src/views/leisure/leisure.view.css

```diff
@@ -62,7 +63,6 @@
 
 .places {
   flex-wrap: wrap;
-  justify-content: start;
   justify-content: space-between;
```

Comment:
this can be achieved by using classes
https://vuetifyjs.com/ru/styles/flex/#flex-wrap
https://vuetifyjs.com/ru/styles/flex/#flex-justify

Reply:
thx